src/main/java/com/alkemy/ong/service/impl/ActivityServiceImpl.java

```diff
@@ -26,4 +28,15 @@ public ActivityDTO save(ActivityDTO dto) {
             Activity savedActivity = activityRepository.save(activity);
             return activityMapper.activityToActivityDTO(savedActivity);
     }
+
+    @Override
+    public List<ActivityDTO> getAll(){
+        List<Activity> activityList = activityRepository.findAll();
+        List<ActivityDTO> dtoList = new ArrayList<>();
+        for (Activity activity : activityList){
```

Comment:
Solo para que lo sepas nada mas, esto se podría haber resuelto con una expresión lambda, pero no hace falta que modifiques nada, es para que tengas otra opción. Quedaría algo como:
```
activityList.forEach{ activity -> 
      ActivityDTO dto = activityMapper.activityToActivityDTO(activity);
      dtoList.add(dto);
}
```
Esta es una de las opciones, hay mas y quizás optimizando mejor, lo que te puse yo es simplemente como hacer un for each con una expresión lambda, lo mismo que hiciste vos con el for each tradicional